build/build-image/Dockerfile

```diff
@@ -34,7 +34,7 @@ ENV HOME=/go/src/k8s.io/kubernetes
 WORKDIR ${HOME}
 
 # Make output from the dockerized build go someplace else
-ENV KUBE_OUTPUT_SUBPATH=_output/dockerized
+ENV KUBE_BUILDNAME=dockerized
```

Comment:
all github refs are just our copy files. so we should be good ( https://github.com/search?type=code&q=%2FKUBE_OUTPUT_SUBPATH%2F )

Reply:
Is this a comment or do I need to do something to resolve this?  Sorry, juggling all the feedback...